src/platform/mhv/secondary-nav/tests/MhvSecondaryNavItem.unit.spec.jsx

```diff
@@ -25,28 +25,57 @@ describe('MHV Secondary Navigation Item Component', () => {
   });
 
   describe('handle abbreviations', () => {
-    it('when provided', () => {
+    it('when provided without arial label', () => {
+      const title = 'a title';
+      const abbr = 'an abbr';
+      const { getAllByText } = render(
+        <MhvSecondaryNavItem
+          title={title}
+          abbreviation={abbr}
+          icon="home"
+          href="/my-health"
+        />,
+      );
+      expect(getAllByText(title).length).to.eql(1);
+      expect(getAllByText(abbr).length).to.eql(1);
+      expect(getAllByText(title)[0].parentNode.getAttribute('aria-label')).to.be
+        .null;
+    });
+
+    it('when provided with aria label', () => {
       const title = 'a title';
       const abbr = 'an abbr';
+      const ariaLabel = 'a label';
       const { getAllByText } = render(
         <MhvSecondaryNavItem
           title={title}
           abbreviation={abbr}
+          ariaLabel={ariaLabel}
           icon="home"
           href="/my-health"
         />,
       );
       expect(getAllByText(title).length).to.eql(1);
       expect(getAllByText(abbr).length).to.eql(1);
+      expect(
+        getAllByText(title)[0].parentNode.getAttribute('aria-label'),
```

Comment:
Did you considered `getByLabelText('a label');` or `getByRole('link', { name: 'a label'})`? I believe those two methods are preferred for looking up elements by accessibility properties, with `getByRole` being the [highest priority query](https://testing-library.com/docs/queries/about#priority). 

[The docs for https://testing-library.com/docs/queries/bylabeltext](https://testing-library.com/docs/queries/bylabeltext) make it sound like its suited for aria-labels in addition to non-aria labels.

Reply:
I did not know getByLabelText would do that. Let me try that out.

Reply:
Updates tests